src/routes/store.svelte

```diff
@@ -37,6 +41,28 @@
   let orderLabel = orderLabels[0];
   let currentPage = 1;
 
+  let newArrivals = null;
+  $: {
+    if (currentPage === 1 && $session.notifications != null) {
+      const newArrivalsNotification = $session.notifications.find(
+        notification => notification.type === NotificationTypes.NEW_ARRIVALS,
+      );
+
+      if (newArrivalsNotification != null) {
+        api.json(api.patch(
+          `/notifications/${newArrivalsNotification.id}/read`,
+          { csrfToken: $session.account.csrf_token },
+        )).catch(console.error);
```

Comment:
and maybe `.then` set the is_read property of that notification to update the local variable and hide the red dot if it was the only notification